Jenkinsfile

```diff
@@ -105,14 +105,13 @@ pipeline {
                             mv OTRGui/build/assets build/
                             mv ZAPDTR/ZAPD.out build/assets/extractor/
                             mv README.md build/readme.txt
-                            cd build
-                            7z a soh-linux.7z soh.elf OTRGui assets readme.txt
-                            mv soh-linux.7z ../
+
+			    docker exec sohcont appimage/appimage.sh
                             
                             '''
                         }
                         sh 'sudo docker container stop sohcont'
-                        archiveArtifacts artifacts: 'soh-linux.7z', followSymlinks: false, onlyIfSuccessful: true
+                        archiveArtifacts artifacts: 'SOH-Linux.AppImage', followSymlinks: false, onlyIfSuccessful: true
```

Comment:
We should probably still create a zip that has at least the readme and app image in it.

Reply:
This is a point to discuss. All of the assets and readme are packaged in the AppImage. 
It only regenerates the OTR if it is not present, because, in my opinion, it would be annoying to regenerate everytime you run the game. There is a message that alerts the user if no ROM is present, or if one that doesn't match the currently accepted hashes. I could do something like display readme on first run.

Reply:
That's fine.. but there are times when users should regenerate their OTR. What do we want to recommend in this case? I guess we could also recommend that they delete the OTR as the launcher will regenerate it. That would work too actually!